src/compiler/types/generate-app-types.ts

```diff
@@ -19,7 +19,7 @@ import { updateReferenceTypeImports } from './update-import-refs';
  * @returns `true` if the type declaration file written to disk has changed, `false` otherwise
  */
 export const generateAppTypes = async (
-  config: d.Config,
+  config: d.ValidatedConfig,
```

Comment:
Verified all the callers of this fn are calling it with a `ValidatedConfig` instance. 👍 👍